spec/pdfs/donation_pdf_spec.rb

```diff
@@ -65,4 +77,22 @@
       expect(pdf_test.page(1).text).to include("Total Items Received")
     end
   end
+
+  context "product drive donation" do
+    it "renders correctly" do
+      pdf = described_class.new(organization, product_drive_donation)
+      pdf_test = PDF::Reader.new(StringIO.new(pdf.compute_and_render))
+      expect(pdf_test.page(1).text).to include(product_drive_donation.product_drive_participant.business_name)
```

Comment:
I much prefer explicit checks - since you've given the participant a specific business name, you should be checking against that string rather than going back to the model you created.

Reply:
Updated! [Refactoring specs to use explicit checks](https://github.com/rubyforgood/human-essentials/pull/4831/commits/dd98442b16ea8edfdf706aa8d75bb14a6b397dde)